libs/application/templates/parental-leave/src/fields/InReviewSteps/ReviewSection/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from 'react'
```

Comment:
Won't this component be also used in others application systems and should be moved to some shared lib?

Reply:
Yes post beta we will discuss this with the design system team and the other application teams. But for now we wanted to do less impact and just add it to the Parental Leave project. 

To make it more re-usable across projects we'll have to make some updates to the state machine and how/if it knows about completed states.